packages/loaders/src/elements/Inline.tsx

```diff
@@ -20,8 +20,9 @@ const InlineTypingIndicator: React.FC<IStyledTypingSvgProps> = props => {
 };
 
 interface IInlineProps extends React.HTMLAttributes<SVGSVGElement> {
-  /* Width of the loader in px */
+  /** Sets the width in pixels and scales the loader proportionally */
   size?: number;
+  /** Sets the fill `color`. Inherits the parent font `color` by default */
```

Comment:
The use of code backticks feels a bit inconsistent. I think they make sense where we are explicitly referring to a CSS property – less so when there is an implicit reference. To mitigate, I'd suggest the following:

"Sets the fill _color_. Inherits the parent font `color` by default"

Reply:
I like this way better.